CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## not released yet
+## v1.2.0 - 05 Nov 2020
+
+With this release, `s5cmd` automatically determines region information of destination buckets and creates reusable sessions.
```

Comment:
Since we always resused AWS sessions, I think we can remove the last part of the sentence.

Reply:
done